README.md

```diff
@@ -6,6 +6,13 @@
        src="https://scan.coverity.com/projects/12056/badge.svg"/>
 </a>
 
+## Getting Started
+Launch `Terminal` in Ubuntu 16.04
+```
+$ sudo apt-get install git
+$ git clone https://github.com/01org/camera-streaming-daemon.git
+```
```

Comment:
I'd rather just point where the git repository is,  something like:

```
Git Repository: https://github.com/01org/camera-streaming-daemon
```

In the section above.